src/events/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as EventBus } from './EventBus';
+export { captureRejectionSymbol } from 'events';
```

Comment:
This should be ahead of the above export. Always export native packages first.

Reply:
Fixed